### PR TITLE
chore: Fix grammar issues in README

### PR DESCRIPTION
**Pull request by mahozad**
**What kind of change does this PR introduce?**
Misc. grammar fixes in the main README file.

**Did you add tests for your changes?**
No.

**If relevant, did you update the documentation?**
No.

**Summary**
- Use the proper article `an` instead of `a`
- Use third-person verb (ending in `s`) instead of base verb

**Does this PR introduce a breaking change?**
No.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Most things that you can do manually in the browser can be done using Puppeteer!
 Here are a few examples to get you started:
 
 - Generate screenshots and PDFs of pages.
-- Crawl a SPA (Single-Page Application) and generate pre-rendered content (i.e.
+- Crawl an SPA (Single-Page Application) and generate pre-rendered content (i.e.
```
Comment (Lightning00Blade) [inline README.md:24]: This is incorrect the prefix for SPA should be `a` as the first letter is not a vowel. Please remove this change.

Comment (mahozad) [reply README.md:24]: No. For words like "studio" we say "*a* studio". For the "s" letter we say "**an** SUV".
  - https://oneminuteenglish.org/en/an-s-or-a-s-which-is-correct/
  - https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an
  - https://www.pristineword.com/grammar-an-s/

Comment (Lightning00Blade) [reply README.md:24]: Yes you are correct there are exceptions but I have only heard it pronounce like the word `spa` [`/spɑː/` [info](https://dictionary.cambridge.org/dictionary/english/spa?q=SPA) ] with a constant start.  
The `SUV` has a pronunciation of `/ˌes.juːˈvi/` which has a vowel start.

Comment (mahozad) [reply README.md:24]: [MDN uses "**an** SPA"](https://developer.mozilla.org/en-US/docs/Glossary/SPA#:~:text=An%20SPA%20(Single%2Dpage%20application)).

Please close this pull request if you think "a spa" is good.

**Comment by Lightning00Blade** [comment]
As per the last comments closing this one.

**Pull request closed without merging**